servers/curated-corpus-api/src/database/mutations/SectionItem.ts

```diff
@@ -30,6 +36,23 @@ export async function createSectionItem(
     );
   }
 
+  // Check if ML is trying to add an item that was manually removed
+  if (createSource === ActivitySource.ML) {
+    const manuallyRemovedItem = await db.sectionItem.findFirst({
+      where: {
+        approvedItemId: approvedItem.id,
+        active: false,
+        deactivateSource: ActivitySource.MANUAL,
+      },
+    });
+
+    if (manuallyRemovedItem) {
+      throw new ForbiddenError(
+        `Cannot create section item: This item was previously removed manually and cannot be re-added by ML.`,
+      );
+    }
```

Comment:
The section manager Lambda will throw a Sentry error when this occurs, and will continue to process the remaining section items.

We can set an ignore rule in Sentry to only get errors after this happens a certain number of times per hour.